tests/headers/macros/macro_conditionals.h

```diff
@@ -50,7 +50,7 @@
 #ifdef UNDEFINED
   #define NO_DEFINE_ELIF
   int NO_DECLARE_ELIF;
-#elif defined MACRO
+#elif defined(MACRO)
```

Comment:
Why ? is that incorrect ? According to https://en.wikipedia.org/wiki/C_preprocessor it is a valid syntax, so we should keep it.